app/assets/stylesheets/views/_landing_page/quote.scss

```diff
@@ -0,0 +1,27 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-b-quote {
+  color: govuk-colour("white");
+
+  .app-b-quote__svg {
```

Comment:
Nitpick: if we're using BEM like this, I don't think we need to nest all these classes.

Reply:
Yep good point. Updated now